src/List/Zipper.elm

```diff
@@ -243,3 +243,25 @@ isFirst (Zipper ls _ _) =
 isLast : Zipper a -> Bool
 isLast (Zipper _ _ rs) =
     List.isEmpty rs
+
+
+{-| Make zippers focused at every element of the input list:
+
+    openAll [1,2,3]
+    -->
+    [ [<1>, 2, 3]
+    , [1, <2>, 3]
+    , [1, 2, <3>]
+    ]
+
+-}
+openAll : List a -> List (Zipper a)
```

Comment:
This seems like a rather inefficient implementation. Wouldn't iterating through it (recursively) like `find` does be a lot more efficient?

Reply:
Reimplemented with recursion, added a benchmark in 60a5e9e. Thanks for the idea!

![Screenshot 2022-09-27 at 12 54 10](https://user-images.githubusercontent.com/149425/192507542-5e610299-2e22-49f7-a89b-33fdd242c74a.png)